src/edu/csus/ecs/pc2/ui/team/QuickSubmitter.java

```diff
@@ -143,7 +164,8 @@ public int sendSubmissions(List<File> filesToSubmit) {
     private Problem guessProblem(IInternalContest contest2, String absolutePath) {
         Problem[] problems = contest.getProblems();
         for (Problem problem : problems) {
-            if (absolutePath.indexOf(problem.getShortName()) != -1) {
+            String problemPath = IContestLoader.CONFIG_DIRNAME + File.separator + problem.getShortName() + File.separator;
+            if (absolutePath.indexOf(problemPath) != -1) {
                 return problem;
             }
         }
```

Comment:
In method `sendSubmission(File sourceFile)`, it calls both `guessLanguage()` and `guessProblem()`.  Both of these methods can return null (if they're unable to find a corresponding language/problem).  Method `sendSubmission(File)` checks for a return value of `null` from `guessLanguage()`, and appropriately logs an error and then returns. 
However, when method `sendSubmission(File)` calls `guessProblem()`,  _it does not check for a null return value_.  I think the code in `sendSubmission(File)` needs to log an error in that condition just like it does when `guessLanguage()` returns null.

(I'm putting this comment here because GitHub won't allow me to comment on the actual line in `sendSubmission(File)`... probably because it is "unchanged" from a previous version of the file.)

I consider this (fixing `sendSubmission(File)` so it checks for a null return from `guessProblem()`) an approval showstopper.